spec/send_sms_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'send_sms.rb'
+
+describe Twillio do
+  subject(:twil)  { described_class.new }
+  let(:order) { double(:order)}
+
+  # before do
+  #   allow(twil).to receive(:confirmation_text)
+  # end
+
+  it 'allow customers to place order by passing current dishes/quantities and expected total' do
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)